### PR TITLE
Implement blacklist feature

### PR DESCRIPTION
**Pull request by Perry3D**
To avoid too much traffic this feature implements a blacklist.

You can add topic names to the `blacklist.json` file as a list. 

For an example see the readme file.

### CODE REVIEW

File: ecal_foxglove_bridge/ecal_foxglove_bridge.py [hunk 1]
```diff
@@ -317,5 +325,13 @@ def parse_arguments():
     args = parser.parse_args()     
     return args
 
+
+def readBlacklist() -> list:
+    with open('blacklist.json', 'r') as f:
```
Comment (KerstinKeller) [inline ecal_foxglove_bridge/ecal_foxglove_bridge.py:330]: Can you make this a parameter? 
So you can do `--blacklist=blacklist.json`, so if it is not present, it doesn't throw an error?

Comment (Perry3D) [reply ecal_foxglove_bridge/ecal_foxglove_bridge.py]: I added an argument for the file and handled a non existing file. It will just put out a warning.

**Comment by Perry3D** [comment]
Ok, i just found a bug when removing subscriber.

**Comment by Perry3D** [comment]
> Ok, i just found a bug when removing subscriber.

Should be fixed with 1afa6d204bfb14308914a7e58f96a407fc6f93b6

**Comment by Perry3D** [comment]
Now the behavior is:
- No blacklist argument specified -> Don't load any blacklist
- Correct blacklist file specified -> Load blacklist
- Wrong blacklist file specified (not found or decode error) -> Exit with error message.

**Review by KerstinKeller — APPROVED** [review]
Looks good now.

**Pull request merged**